app/src/main/java/org/systers/mentorship/view/activities/MainActivity.kt

```diff
@@ -35,7 +35,7 @@ class MainActivity : BaseActivity() {
         if (savedInstanceState == null) {
             showHomeFragment()
         } else {
-            atHome = savedInstanceState.getBoolean("atHome")
```

Comment:
this type of string does not need to be in string resources, this is not exposed to the user, its an internal string used for piece of logic. This would be more suited for a constant probably 🤔

Reply:
> this type of string does not need to be in string resources, this is not exposed to the user, its an internal string used for piece of logic. This would be more suited for a constant probably 🤔

Okay all the occurrences of "getString(R.string.at_home)" will be changed to "atHome".